python_scripts/functions.py

```diff
@@ -74,16 +71,20 @@ def match_files_in_land_to_config(config):
             ]
         else:
             table_params["matched_files"] = [
-                lf for lf in land_files if lf.replace(land_base_path, "").startswith(table_name)
+                lf
+                for lf in land_files
+                if lf.replace(land_base_path, "").startswith(table_name)
```

Comment:
minor thing, `lf` looks all too similar to `if`

Reply:
Changed.